web/src/pages/apd/activities/contractor-costs/ContractorResource/ContractorResourceForm.js

```diff
@@ -16,16 +16,16 @@ import { saveContractor as actualSaveContractor } from '../../../../../redux/act
 
 const getCheckedValue = value => {
   if (value !== null) {
-    if (value === true) return 'yes';
-    if (value === false) return 'no';
+    if (value === true) return 'true';
```

Comment:
this is a great improvement but I wonder why not have it remain a bool throughout the system? is it because the design system component doesn't support using a 'bool' as a value?

Reply:
I'm pretty sure I tried using just the booleans and it didn't like it, but there's no reason it shouldn't so I'll try it again

Reply:
the design system wants to return 'true' or 'false' even if the value is a boolean, but I was able to work around that